README.md

```diff
@@ -3,60 +3,70 @@
 
-    helm install --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+```
+helm install [--set storageClass=mystorageclass] --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+``` 
 
-It is very important that the `--wait` is included as the chart uses a post-install hook to complete the installation.  During the installation, you will see the engine-controller, metrics and resource-monitor pods restart a few times, this is expected as they cannot start until the api pod has correct initialised and the api pod cannot start until the etcd and couchdb have completed their startups. 
+To install the latest development version of the Galasa Ecosystem chart, replace `galasa/ecosystem` in the above command with the path to the [`ecosystem`](./charts/ecosystem) directory in this repository.
 
-The `galasaVersion` value is the version of Galasa you want to run with.  You should not use latest to ensure each pod in the Ecosystem is running at the same level.
+The `--wait` flag ensures the chart installation has completed before marking it as "Deployed". During the installation, the API pod waits for the etcd and RAS pods to initialise while the engine-controller, metrics, and resource-monitor pods wait for the API pod to initialise.
 
-The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services.
+The `galasaVersion` value is the version of Galasa you want to run. You should not use latest to ensure each pod in the Ecosystem is running at the same level.
 
-The Galasa Ecosystem Helm chart will deploy three persistent volumes.  If you need to provide a Kubernetes storage class for these PVs, then please use the following command:-
+The `externalHostname` value is the DNS hostname or IP address of the Kubernetes node that be used to access the Galasa NodePort services. If you are deploying to minikube, this can be retrieved by running `minikube ip`.
 
-    helm install --set storageClass=mysgtorageclass --set galasaVersion=0.23.0 --set externalHostname=my.host.name <release-name> galasa/ecosystem --wait 
+After the Ecosystem has been successfully deployed, you can run the following command to ensure the Ecosystem can be accessed externally to Kubernetes and a simple test engine can be run:
 
-After the Ecosystem has been successfully deployed, you can run the following command to ensure the Ecosystem can be accessed externally to Kubernetes and a simple test engine can be run:-
+```
+helm test <release-name>
+```
 
-    helm test <release-name>
+Once the Ecosystem is up and running, you will need to determine the URL of the Ecosystem bootstrap. Issue the command:
```

Comment:
How can I tell if the ecosystem is up and runnning ?
Once I've successfully tested with help text ?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/1/commits/fad3b94c4581df0dae003f50aac19569f88aa2f1